src/urllib3/response.py

```diff
@@ -60,6 +66,9 @@ def decompress(self, data):
             finally:
                 self._data = None
 
+    def flush(self) -> bytes:
+        return self._obj.flush()
```

Comment:
Wow, this is great! Much more explicit and actually less characters. The magic `__getattr__` always confused me